src/server/adminBro.ts

```diff
@@ -481,6 +481,9 @@ const getAdminBroInstance = () => {
                 edit: false,
               },
             },
+            isImported: {
+              isVisible: { list: false, filter: false, show: true, edit: true },
+            },
```

Comment:
As `isImported` is a variable that depends on `traceCampaignId` and `givingBlockId`,.. I think we should not add it in edit page , because we should not give this option to admins to can change it because it might cause some inconsistency in our system